src/app/public/modules/country-field/types/country.ts

```diff
@@ -0,0 +1,18 @@
+export interface SkyCountryFieldCountry {
```

Comment:
Should we have a jsdoc descriptor on the interface?

Reply:
Done